packages/browser/src/Component/Widget/Server/SPARQL/View/SparqlQueryWindow.css

```diff
@@ -1,78 +1,96 @@
-#_window_sparqlQueryWindow {
-  width: 500px;
-  max-width: 100%;
-  max-height: 100%;
-}
-
-#_window_content_sparqlQueryWindow {
-  overflow: hidden;
-}
-
-#_window_inner_content_sparqlQueryWindow {
-  overflow: inherit;
-}
-
-#_window_sparqlQueryWindow_form {
-  width: 100%;
-  height: 100%;
-}
-
-#_window_sparqlQueryWindow_query_text_area {
-  width: 100%;
-  height: 100%;
-  min-height: 150px;
-  resize: vertical;
-  box-sizing: border-box;
-}
-
-#_window_sparqlQueryWindow_query_button {
-  width: 200px;
-  height: 32px;
-}
-
-#_window_sparqlQueryWindow_data_view {
-  width: 100%;
-  height: 100%;
-  background-color: rgba(218, 218, 218, 0.08);
-}
-
-#_window_sparqlQueryWindow_filter{
-  position: fixed;
-  padding: 1px;
-  background-color: rgba(30,30,30,0.7);
-}
-
-table {
-  border: 1px solid white;
-  width: 100%;
-  height: 100%;
-  margin-top:24px !important;
-}
-
-thead {
-  color: #90EE90;
-  margin:auto;
-  border: 1px solid white !important;
-}
-
-tr {
-  border: 1px solid white !important;
-}
-
-td {
-  border: 1px solid white !important;
-}
-
-th.des:after {
-  content: "\21E9";
-}
-
-th.aes:after {
-  content: "\21E7";
-}
-
-.renderjson {
-  margin: 8px;
-  height: 500px;
-  overflow: scroll;
-}
\ No newline at end of file
+#_window_sparqlQueryWindow {
+  width: 500px;
+  max-width: 100%;
+  max-height: 100%;
+}
+
+#_window_content_sparqlQueryWindow {
+  overflow: hidden;
+}
+
+#_window_inner_content_sparqlQueryWindow {
+  overflow: inherit;
+  font-size: 20px;
+  font-weight: bold;
+}
+
+#_window_sparqlQueryWindow_form {
+  width: 100%;
+  height: 100%;
+}
+
+#_window_sparqlQueryWindow_query_select {
+  margin-bottom: 2px;
+}
+
+#_window_sparqlQueryWindow_query_text_area {
+  width: 100%;
+  height: 100%;
+  margin-top: 4px;
+  margin-bottom: 4px;
+  resize: vertical;
+  box-sizing: border-box;
+}
+
+#_window_sparqlQueryWindow_toggle_query_text_area_button {
+  background: none;
+  border: 0;
+  padding: 1px 4px 1px 4px;
+  margin: auto;
+  color: white;
+}
+
+#_window_sparqlQueryWindow_query_button {
+  width: 200px;
+  height: 32px;
+}
+
+#_window_sparqlQueryWindow_data_view {
+  width: 100%;
+  height: 100%;
+  font-size: 12px;
+  font-weight: normal;
+  background-color: rgba(0, 0, 0, 0.2);
+  border-radius: 10px; 
+}
+
+#_window_sparqlQueryWindow_filter{
+  position: fixed;
+  padding: 1px;
+  background-color: rgba(30,30,30,0.7);
+}
+
+table {

```

Comment:
don't apply css style to global html write something like
.root_class table{
}

#root_id table{
}

to select child table of the sparql widget

Reply:
fixed